Add initial set of questions (#31)
huserben/scrum-kata: questions/skill_acquisition_in_the_product_backlog.md

---
layout: post
title: "Skill Acquisition in the Product Backlog"
parent: Katas
---

> If the team lacks a skill, is it okay to have items about acquiring that skill (e.g. training, hiring etc.) in the Product Backlog? As this is needed to improve the Product?

Reviewer: ```suggestion
If the team lacks a skill, is it okay to have items about acquiring that skill (e.g. training, hiring etc.) in the Product Backlog assuming it's needed to improve the Product?
```
